From 2f1a35162f57ed81467c175237228b0830416c3f Mon Sep 17 00:00:00 2001
From: trainer <trainer@training.com>
Date: Mon, 9 May 2022 12:28:11 +0000
Subject: [PATCH] testing git patch

---
 test.c | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 test.c

diff --git a/test.c b/test.c
new file mode 100644
index 0000000..47e645d
--- /dev/null
+++ b/test.c
@@ -0,0 +1 @@
+Testing git patch
\ No newline at end of file
-- 
2.36.0.windows.1

